src/pages/steps/AdditionalInformationStep/DescriptionStep.tsx

```diff
@@ -85,7 +92,11 @@ const DescriptionStep = ({
   // TODO: refactor
   const eventId = offerId;
 
-  const [description, setDescription] = useState('');
+  const [editorState, setEditorState] = useState(EditorState.createEmpty());
+  const plainTextDescription = useMemo(
+    () => editorState.getCurrentContent().getPlainText(),
+    [editorState],
```

Comment:
is editorState a stable reference? because if it is a regular un-reffed object ti will cause the `useMemo` to never memoize the value.

Reply:
Yeah true need to check that 🤔

Reply:
Actually I think it's fine like this. The fact that it is initialized with `useState` and not a `const` with an object value.

Never mind my comment 😉

Reply:
Ok I just checked and because it's an immutable structure it does seem to be able to memoize it properly even when the rest of the component changes 👍

Reply:
I tested it quickly in [codesandbox](https://codesandbox.io/s/nifty-mountain-qq1r3g?file=/src/App.js).

Since the object is a value returned by `useState` it won't retrigger the `useMemo` when other state is changed.
When the object is declared as a plain `const` it will. So we're fine here I think 👍🏻